### DISCUSSION TITLE
My account shows no domains

### DISCUSSION THREAD

**Discussion by JohannesSanders** [vercel/community#6872] [Account Issue] [answered]
### Question

I have bought several domains with Vercel, but I don't have the option to edit them. The domains tab is empty, and it asks me to add or move a domain, even though I bought several before.

**Comment by kubaprzetakiewicz** [comment]
same, but i did get a notification about a domain renewal which, when clicked, results in a state pictured below

<img width="1266" alt="Screenshot 2024-05-21 at 21 20 04" src="https://github.com/vercel/community/assets/18512701/a672a12c-7ba9-49df-b62a-ce4c5215bf84">
<img width="933" alt="Screenshot 2024-05-21 at 21 20 09" src="https://github.com/vercel/community/assets/18512701/6893950d-f72b-4322-a331-1c95592a7d51">

storage was cleared out, no broken local storage, cookies, etc

console gives me a 403 and a 429
```
[Log] Ignored request error – CustomFetchError: You don't have permission to list the domain record. (8756.ab5bc6ca46282428.js, line 11)
CustomFetchError: You don't have permission to list the domain record.
```

**Reply by szul** [reply]
I am getting the same thing. No domains listed. Renewals in the notification, but when I click on the renewal, I get a 404 error.

**Reply by Adron** [reply]
Came here with the exact same issue. No domains listed even though I'd transferred a slew of them for upcoming projects. Came to work on em' and nothing is there. I however got no notification, no update, no details about any of them disappearing so I've no idea what happened to them. I checked the registrar and they seem to still be here.

**Comment by amyegan** [comment]
When a domain is purchased or transferred in, it is automatically associated with your personal account. Please check the list of domains belonging to your **account** at [vercel.com/account/domains](https://vercel.com/account/domains) and let me know whether or not you find the domains there.

If the domains are currently associated with your personal account, you can move them to your preferred team using these steps: [vercel.com/docs/projects/domains/transfer-your-domain](https://vercel.com/docs/projects/domains/transfer-your-domain#)

**Reply by lorenzejay** [reply]
this worked! How do we see our personal account from the dash? This is unclear how to get to

**Reply by amyegan** [reply]
You can see your personal account info by clicking on your avatar in the top right corner and then choosing **Account Settings**. From there you can manage your display name, email, authentication options, domains, and more.

**Reply by kubaprzetakiewicz** [reply]
the link from notifications still does not work though and results in 403

https://vercel.com/dashboard/domains/<domain_name>

**Reply by trevor-e** [reply]
This just happened to me as well and was incredibly confusing. I had a bunch of domains that were associated with my personal account and were displaying fine. Then I went and bought a new domain, the first since having my account migrated to a personal hobby team. Suddenly all of my old domains were gone and only the most recently I just bought showed up. Having to find them in my personal settings, when they previously showed up just fine, is not intuitive at all.

**Reply by amyegan** [reply]
Thank you all for sharing your experiences. 

We're working on an update to make this easier which will be available very soon.

**Reply by trevor-e** [reply]
New fix looks great! Thanks.

<img width="438" alt="Screenshot 2024-06-08 at 9 06 32 PM" src="https://github.com/vercel/community/assets/1447798/22bd904e-d0e2-4fe7-88ff-b1218bbb69d1">

**Comment by amyegan** [comment, marked as answer]
We made a change so you can see your account domains listed alongside your teams' domains! You can see an example shared in https://github.com/vercel/community/discussions/6872#discussioncomment-9716917 or by visiting your own team's domain page now.

https://vercel.com/changelog/account-owned-domains-now-visible-in-team-scope-domains-tab

Please let me know if you have questions or run into any issues with this change.

**Reply by Adron** [reply]
Excellent! That's the intuitive place I'd assume they'd be. Thanks to the team! 🤘🏻
